README.md
# Song and Silence: A Mod for Bards and Thieves
[A Gibberlings Three Mod][Gibberlings Three]

**Authors**: [Andyr], [Grim Squeaker], and [NiGHTMARE]  
**On the Web**: [Home page] and [discussion forum]

**Languages**: English, French, German, Polish, and Russian
**Platforms**: Windows, Mac OS X, and Linux

## Overview

This mod adds many options to those who use thieves and bards in Baldur's Gate: Enhanced Edition, Baldur's Gate II: Enhanced Edition, BG2: Shadows of Amn (with or without the expansion pack Throne of Bhaal) and with the mod conversions Baldur's Gate Trilogy and Baldur's Gate Tutu. For bug reports and suggestions please visit the [discussion forum].


## Compatibility

This mod is designed to work with Baldur's Gate: Enhanced Edition, Baldur's Gate II: Enhanced Edition, BG2: Shadows of Amn (with or without the expansion pack Throne of Bhaal) and with the mod conversions Baldur's Gate Trilogy and Baldur's Gate Tutu. The mod is fully compatible with the changes made in the Ashes of Embers "Universal Weapons" component, provided Song and Silence is installed after it. The Rogue Rebalancing and Refinements mods from [Spellhold Studios] also alter rogues (bards and thieves); if you are using these then you should get the benefits of whichever was installed most recently. Until I get HLAs (High Level Abilities) done in this mod I would recommend installing one of them before this mod, so you get their HLAs and my new kits and changes to classes. All the new kits in this mod currently use the standard HLA tables so will be compatible with Refinements and Rogue Rebalancing. If you have kitted Imoen or Nalia with the Adventurer kit from the NPC Kitpack, installing Song and Silence's Adventurer kit will update them to the (slightly different) S&S version of thhe Adventurer.

## Contents

Each component is separate and optional. The list of components is as follows:

### Changes to trueclass bards and thieves, and unmodded game kits (required for other components)
This is required for all other components.

This component changes thieves and bards to be more in line with P&P rules and my own personal views on the classes. Firstly, alignment restrictions have been changed. Thieves can now be of any alignment (previously Lawful Good was prohibited) as can kitless Bards and Blades. Jesters and Skalds can now be of any non-Lawful alignment.

Secondly, this component fixes several bugs to do with bards and rogues in the unmodded game. These are listed below. Some of these bugs may be fixed by other mods/patches/fixpacks such as [BG2 Fixpack]. There will be no problems with having the fixes from both sources installed.

- Blades not actually suffering a penalty to their Pick Pockets skill as the description states.
- Skalds were not getting a bonus to hit as the description states.
- Swashbucklers had incorrect maximum proficiency slots in Katana, Crossbow, Short Bow and Sling.

Thirdly, thieves (kitless and kitted) can now place 3 proficiency points in Two Weapon Fighting Style. The Swashbuckler description has been altered to reflect the fact that this is now not unique to them.

Further versions will include HLA changes - at the moment I would suggest using aVENGER's Rogue Rebalancing from [Spellhold Studios] as an alternative. aVENGER has graciously allowed us to use his work as a basis for HLAs in this mod, so if you've been using Rogue Rebalancing and liked that you should hopefully enjoy this mod too!

### Add new bardic store and thief items
This component adds a new storekeep, Raoul, to the game who sells a variety of interesting new items. These are mainly for bards but other classes may find something to suit as well... So be sure to take a peek. Raoul can be found standing between two tents at the Nashkel Carnival in BGEE/BGT/Tutu, and outside the Five Flagons at the Merchant's Square in the Athkatla Bridge District in BGIIEE/BGT/BG2. Note that while Raoul will appear in both the Carnival and the Bridge District in BGT games, his store inventory will not carry over from Nashkel to Athkatla.

A selection of new items designed with thieves in mind has also been added; in BGEE/BGT/Tutu they can be found in the inventory of Black Lily (in the Thieves' Guild in the city of Baldur's Gate) and in BGIIEE/BGT/BG2 you can find them with Roger the Fence (in the Sewers of Athkatla underneath the Temple District).

### Install Acrobat bard kit

> ACROBAT: As entertainers, most Acrobats are typically found accompanying carnivals or circuses, although a troupe of Acrobats can put on a remarkable show all by themselves. The often perform in large taverns or in clubs, using their arts to draw in the crowd and captivate them long enough for the proprietor to drain their purses. Acrobats are flashy and love to show off their skills; whatever they do, they like to do with a sense of style and a flair for the dramatic. Acrobats sometimes become cat burglars extraordinaire, and they are also useful adventuring allies, as they can squeeze into places where others cannot go without the aid of magic. Their skill at avoiding traps and other dangers also makes them excellent scouts.
> 
> Advantages:
> - +1 bonus to Armor Class
> - +1 bonus to Armor Class vs. missiles every 8 levels
> - May place up to 2 proficiency points in slings, darts and daggers
> 
> Disadvantages:
> - May only place proficiency points in weapons a thief can use
> - May not wear armor heavier than Studded Leather
> - Only has one-half normal lore value

### Install Chorister bard kit

> CHORISTER: Music, whether hymn, symphony or dark chanting intonation, plays an important part in the services of most deities. For aiding in performing rituals or just leading the faithful in worship, most holy places have at least one bard, known as a Chorister.
> 
> Choristers are not only to be found within temple walls, however; many faiths use them as missionaries, helping to bring new worshippers into the fold. Many an adventuring bard with a religious bent can also end up as a Chorister, finding their prayers answered by the deity they venerate.
> 
> Advantages:
> - Knows a small selection of divine spells. These are accessed and memorised as Mage spells:
> Level 1: Armor of Faith, Cure Light Wounds, Doom
> Level 2: Chant, Draw Upon Holy Might, Slow Poison
> Level 3: Cure Medium Wounds, Glyph of Warding
> Level 4: Defensive Harmony, Holy Power
> Level 5: Mass Cure
> Level 6: Bolt of Glory
> 
> Disadvantages:
> - Only has one-half normal Lore value
> - Only has one-half Pick Pockets percentage

### Install Dirgesinger bard kit

> DIRGESINGER: Dirgesingers voice melodies not of celebration and joy, but of sorrow and grief. They seek to spread this melancholy outlook far and wide, believing that only those who give in to their sadness can truly understand the world.
> 
> These sad wanderers seek to express their grief through songs that teach the hearts of their listeners the meaning of true sorrow. Some of these fallen bards want nothing more than for others to understand the depths of their loss. A few are sinister creatures who believe that, since joy has been extinguished for them, they must in turn extinguish the joy of others by using their powers to teach the folly of love, the futility of hope, and the finality of the grave. Dirgesingers of this last sort often associate themselves with powerful undead; serving in the courts of vampire lords or lich-kings.
> 
> Dirgesingers are limited to non-good alignments.
> 
> Advantages:
> - Song Curses enemies, giving them a -1 penalty to saves and THAC0. At level 10 the penalty becomes -2, at level 20 -3 and level 30 -4.
> - From level 5 may use a Song of Horror 1/day. This has the same effects as the level 2 Wizard spell, Horror
> - From level 7 may use a Song of Grief 1/day. This has the same effects as the level 4 Wizard spell, Confusion
> - From level 9 may use a Song of Awakening 1/day. This has the same effects as the level 5 Wizard spell, Animate Dead
> 
> Disadvantages:
> - Only has one-half normal Lore value

### Install Gypsy bard kit

> GYPSY: Gypsies are often called "travelling people" or "forest nomads," typically wandering the lands in caravans comprising of brightly painted carts or wagons. These people are well-known for their strange music and dances, typically surviving in their wandering lifestyle by entertaining villagers and folk in the places they roam. Gypsies wander the land, experiencing life as they go. They travel until they find something of interest, where they establish a temporary camp and remain until the urge to see more of the world overpowers their desire to stay. Adventuring gypsies may be on a personal quest to locate some unique place or thing or may simply be out to experience the great wonders that appear along the adventuring path.
> 
> Gypsies are often misunderstood by others, who classify them as thieves, beggars, carnival people, nomads, or any number of other erroneous names. They are also very open and welcoming to those who seek to learn their ways and join them in their wandering, garnering for them a reputation for stealing children in the night and then spiriting them off into the wilderness. It should be noted that not all Gypsies are bards or have bardic abilities; Gypsy bards, known among Gypsy clans as "rawuns," function as the chief storytellers, lore-keepers, fortune tellers, and spiritual advisers of their Gypsy clans.
> 
> Advantages:
> - Song charms enemies unless they save vs. spells at +2, with a -1 penalty per 5 levels of the bard
> - Has a +1 bonus to saves vs. spells per 8 levels
> - Can cast Curse once per day. The opposite of a Bless spell, this gives foes a -1 penalty to THAC0 and saves
> 
> Disadvantages:
> - Cannot learn mage spells from the schools of Conjuration, Invocation or Transmutation

### Install Adventurer thief kit

If you have kitted Imoen or Nalia with the Adventurer kit from the NPC Kitpack, installing this will update them to this version of the Adventurer.

> ADVENTURER: The adventurer is a jack-of-all-trades, not so much a thief as a character who takes advantage of general thiefly skills on adventures. An Adventurer is preferred by many adventuring parties, because he is much less likely than other thieves to betray or steal from his own companions. The successful Adventurer knows the value of trust and cooperation, while many a "street thief" has been raised on duplicity and (sometimes literal) backstabbing.
> 
> Advantages:
> - +5% to Open Locks and Find/Remove Traps every other level
> - Gains a +1 bonus to all saves at levels 1, 8 and 14
> 
> Disadvantages:
> - No backstab multiplier

### Install Burglar thief kit

> BURGLAR: This sort of thief is the consummate Burglar. He is an expert at breaking and entering the most difficult buildings, bypassing walls, locks, traps and guardians, grabbing the best loot, and escaping unnoticed as stealthily as he arrived.
> 
> Advantages:
> - +10% to Find Traps, Hide In Shadows, Move Silently and Open Locks
> - Can Specialize in Daggers, Darts, and Slings
> 
> Disadvantages:
> - -20% to Pick Pockets and Set Traps
> - Gains trap usage at half the rate of a normal thief

### Install Soulknife thief kit

> SOULKNIFE: This sort of thief, a being possessed of potent psionic power, recognises his or her own mind as the most beautiful and deadly thing in all creation. With practice, they have learned to forge their mental strength into a semisolid blade of pure psychic energy. This weapon, known as a mind blade, means that even when unarmed the soulknife is but a thought away from a deadly strike.
> 
> Advantages:
> - Can achieve Mastery in short swords
> - Can manifest a Mind Blade once per day per 10 levels. This short sword is a psionic weapon that lasts for 1 round/level, and counts as being of enchantment 1 higher than it is when determining what it can hit, to a maximum of +5. Each hit heals the Soulknife 1 hit point. THAC0 bonus and damage improve by 1 per 5 levels, to a maximum of +5 at level 25. Range improves by 10 every 5 levels to a maximum of 50 at level 25.
> 
> Disadvantages:
> - Cannot set traps
> - Only gets 15 skill points per level

### Install Sharpshooter thief kit

> SHARPSHOOTER: This sort of thief tends to be a stealthy character, preferring to engage their target from afar with bolt or arrow rather than face them in dangerous melee combat. Unlike the ranger, the Sharpshooter uses her skills to aid in dubious activities involving the property and lives of others rather than the protection of the great outdoors.
> 
> Although well-versed in the use of many missile weapons, sharpshooters are not as resilient as true warriors, and their propensity for missile attacks over melee means that they lack the opportunism of other thieves when it comes to backstabbing. They tend not to see this as a disadvantage, instead striking from the shadows with a well-aimed projectile, or setting a deadly trap for those who would follow, when necessary.
> 
> Advantages:
> - Can attain Grand Mastery in any missile weapon a thief can use
> - May coat a missile weapon in poison once per day per 4 levels. Attacks made with the weapon in the next round will inject the poison into the target, dealing out 1 damage per second for 24 seconds (3 damage for the first 6 seconds). A saving throw vs. poison limits damage to 12 total.
> 
> Disadvantages:
> - No backstab multiplier

### Install Shadowdancer thief kit

Since BGEE/BGIIEE already have their own Shadowdancer, this kit is only available for BG2/Tutu/BGT.

> SHADOWDANCER: Operating in the border between light and darkness, Shadowdancers are nimble artists of deception. They are mysterious and unknown, never completely trusted but always inducing wonder when met. Despite their link with shadows and trickery, Shadowdancers are as often good as evil. They may use their incredible abilities as they wish.
> 
> Shadowdancers often work in troupes, never staying in one place too long. Some use their abilities to entertain. Others operate as thieves, using their abilities to infiltrate past defenses and dupe others. All Shadowdancer troupes maintain an aura of mystery among the populace, who never know whether to think well or ill of them.
> 
> Advantages:
> - Gains a +5% bonus to Hide and Move Silently skills per 2 levels
> - Has permanent Infravision
> - May cast Reflected Image once per day
> - From level 5, can summon a Shadow once per day. The Shadow increases in power as the Shadowdancer gains levels, has stealthy abilities, and is summoned for 8 hours.
> - From level 8, may cast Shadow Door once per day. Further uses are gained at levels 14 and 20.
> 
> Disadvantages:
> - May not set traps
> - Only gains 15 skill points per level

## Contact Information

[Andyr], [Grim Squeaker], and [NiGHTMARE] created this mod. You can visit the [Gibberlings Three forums] for information on this and any other mods on which we may be working.


## Credits and Copyright Information

This mod may not be sold, published, compiled or redistributed in any form without the consent of its authors. The Song and Silence mod is ©2004-2020, Andyr, Grim Squeaker, and NiGHTMARE.

## Further Reading

- [Installation](INSTALL.md)
- [Change Log](CHANGELOG.md)


[Home page]: http://www.gibberlings3.net/sns/
[Discussion forum]: http://gibberlings3.net/forums/index.php?showforum=72

[Andyr]: http://gibberlings3.net/forums/index.php?showuser=30
[Grim Squeaker]: http://gibberlings3.net/forums/index.php?showuser=25
[NiGHTMARE]: http://gibberlings3.net/forums/index.php?showuser=17

[Gibberlings Three]: http://www.gibberlings3.net/
[Gibberlings Three forums]: http://www.gibberlings3.net/forums/
[Spellhold Studios]: http://www.spellholdstudios.net/

[BG2 Fixpack]: http://www.gibberlings3.net/bg2fixpack/


CHANGELOG.md
# Change Log

## Version 10 - August 3, 2020
### Added
- Russian translation (thanks SubZero400 and yota13)

## Version 9 - August 27, 2018
### Added
- Added compatibility for IWDEE

### Fixed
- Fixed bug where Nalia and Imoen would not get the S&S version of the Adventurer kit when used in conjunction with NPC Kitpack
- The bard songs of the Gypsy and Dirgesinger have had their areas of effect upped to match the bardsong of Jesters (15' radius)
- The curse spell of the Gypsy no longer reports 'Bless' on its target
- The Gypsy bardsong can now be resisted by elven sleep/charm resistance (if installed from Fixpack or the EEs); the spell will also no longer apply effects or require saving throws from non-humanoids

## Version 8 - October 3, 2016
### Added
- EET compatibility

### Fixed
- EE item description formatting (en dash prefix in lists)

### Updated
- Updated WeiDU to v240
- Updated Polish translation (thanks Cahir)

## Version 7 - September 15, 2016
### Fixed
- Updated to latest version of fl#add_kit_ee function for compatibility with EE 2.0
- Corrected errors with usage of HANDLE_CHARSETS (thanks Isaya)
- Corrected encoding errors in English .tra files (thanks Isaya)
- Corrected descriptions that displayed duplicated usability info on EE (thanks Isaya)
- Corrected typos in kit and item descriptions

### Updated
- Updated WeiDU to v239
- Updated French translation (thanks Isaya)

### Removed
- Replaced outdated HANDLE_CHARSETS function with the updated one included in WeiDU

## Version 6 - September 25, 2014
### Added
- Raoul will now appear in the Nashkel Carnival for BGT games
- Updated all kits and content to work on BGEE/BGIIEE games

### Updated
- Streamlined install of Raoul's audio

### Removed
- Purged several empty/unused/unnecessary files, consolidated others

## Version 5 - April 30, 2010
### Fixed
- Resolved compatibility issue with Assassin's Poison Weapon patching (thanks aVENGER)
- Fixed Chorister and Dirgesinger armor usability issues (thanks Mike1072)
- Fixed Acrobat scroll and potion usability issues (thanks Mike1072)
- Fixed "Myself" script error causing incorrect targeting (thanks Lollorian)

### Updated
- Updated WeiDU to v216

## Version 4 - July 02, 2009
### Added
- German translation (thanks Roter Berserker)

### Fixed
- Fixed bug causing everyone EXCEPT Nalia and Imoen to be turned into Adventurers (thanks Steve and Mike1072)
- Fixed Summon Shadow spell causing crashes (thanks Mike1072)
- Ensure thieves get 3 points in dual-wielding (thanks Nythrun and Mike1072)
- Fixed typos in readme and .tra files

### Updated
- Updated WeiDU to v211

## Version 3 - August 1, 2008
### Added
- French translation (thanks Zayn and Lothringen of the d'Oghmatiques)
- Polish translation (thanks Yarpen)
- README/VERSION tags

### Fixed
- Fixed cosmetic readme issues
- Fixed bugs and reported problems through June 2008 (thanks Nythrun)

### Updated
- Updated WeiDU to v208

### Removed
- Removed obsolete auto-update.bat

## Version 2 - September 18, 2007
### Added
- Tra-ified, for ease of potential translators
- Incorporated the PnP bard spell progression table
- The mod will now detect Idobek's NPC Kit and use the Adventurer kit from this mod for Nalia and Imoen instead
- Added the Shadowdancer thief kit

### Fixed
- Changed the way Raoul is spawned, as it could interfere with fog-of-war clearance

### Updated
- Updated WeiDU to v200
- Tweaked the songs of the Dirgesinger and Gypsy kits
- Tweaked the Adventurer's abilities
- Increased the cost of Caster's Chain, as it was too cheap
- Given the Sharpshooter a little power boost

## Version 1 - January 5, 2005
### Added
- Initial release... If you have read as far down the ReadMe as this, congratulations. May your loins sire many children. Now go and play.


song_and_silence/items/a!raoul.d
BEGIN ~A!RAOUL~

  IF ~NumTimesTalkedTo(0)~ THEN BEGIN FirstMeet SAY @0
    IF ~~ THEN REPLY @1 GOTO Intro
    IF ~~ THEN REPLY @2 GOTO NoName
    IF ~~ THEN REPLY @3 GOTO Intro
    IF ~~ THEN REPLY @4 GOTO NoName
  END
  
  IF ~NumTimesTalkedToGT(0)~ THEN BEGIN NotFirstMeet SAY @5
    IF ~~ THEN REPLY @6 GOTO Bye
    IF ~AreaCheck("ar0500")~ THEN REPLY @7 DO ~StartStore("A!RAOUL2",LastTalkedToBy())~ EXIT
    IF ~!AreaCheck("ar0500")~ THEN REPLY @7 DO ~StartStore("A!RAOUL",LastTalkedToBy())~ EXIT
    IF ~~ THEN REPLY @8 EXIT
    IF ~AreaCheck("ar0500")~ THEN REPLY @9 DO ~StartStore("A!RAOUL2",LastTalkedToBy())~ EXIT
    IF ~!AreaCheck("ar0500")~ THEN REPLY @9 DO ~StartStore("A!RAOUL",LastTalkedToBy())~ EXIT
    IF ~~ THEN REPLY @10 GOTO Rumours
  END
  
  IF ~~ THEN BEGIN Intro SAY @11
    IF ~~ THEN REPLY @6 GOTO Bye
    IF ~~ THEN REPLY @12 GOTO Wares
    IF ~~ THEN REPLY @8 EXIT
    IF ~~ THEN REPLY @10 GOTO Rumours
  END
  
  IF ~~ THEN BEGIN NoName SAY @13
    IF ~~ THEN REPLY @6 GOTO Bye
    IF ~~ THEN REPLY @12 GOTO Wares
    IF ~~ THEN REPLY @8 EXIT
    IF ~~ THEN REPLY @10 GOTO Rumours
  END

  
  IF ~~ THEN BEGIN Wares SAY @14
    IF ~AreaCheck("ar0500")~ THEN REPLY @15 DO ~StartStore("A!RAOUL2",LastTalkedToBy())~ EXIT
    IF ~!AreaCheck("ar0500")~ THEN REPLY @15 DO ~StartStore("A!RAOUL",LastTalkedToBy())~ EXIT
    IF ~AreaCheck("ar0500")~ THEN REPLY @16 DO ~StartStore("A!RAOUL2",LastTalkedToBy())~ EXIT
    IF ~!AreaCheck("ar0500")~ THEN REPLY @16 DO ~StartStore("A!RAOUL",LastTalkedToBy())~ EXIT
    IF ~~ THEN REPLY @17 EXIT
    IF ~AreaCheck("ar0500")~ THEN REPLY @18 DO ~StartStore("A!RAOUL2",LastTalkedToBy())~ EXIT
    IF ~!AreaCheck("ar0500")~ THEN REPLY @18 DO ~StartStore("A!RAOUL",LastTalkedToBy())~ EXIT
    IF ~~ THEN REPLY @19 GOTO Rumours
  END
  
  IF ~~ THEN BEGIN Rumours SAY @20
    IF ~!AreaCheck("ar0500")~ THEN GOTO rumor_bg
    IF ~AreaCheck("ar0500")~ THEN GOTO rumor_bg2
  END
  
  IF ~~ THEN BEGIN rumor_bg2 SAY @21 = @22 = @23 = @24
    IF ~~ THEN GOTO rumor_end
  END
  
  IF ~~ THEN BEGIN rumor_bg SAY @27 = @28 = @29 = @30
    IF ~~ THEN GOTO rumor_end
  END
  
  IF ~~ THEN BEGIN rumor_end SAY @25
    IF ~~ THEN EXIT
  END
  
  IF ~~ THEN BEGIN Bye SAY @26
    IF ~~ THEN EXIT
  END
